src/lecturer/components/ListView/ListView.tsx

```diff
@@ -0,0 +1,203 @@
+            position: "absolute",
+            width: "100%",
+            top: 0,
+            zIndex: -1,
+            paddingTop: 75,
+            paddingBottom: "10px"
+        },
+        cardWrapper: {
+            position: "relative",
+        },
+        list: {
+            backgroundColor: theme.palette.background.paper,
+            width: "500px",
+            height: "500px",
+            overflow: "auto",
+        },
+        bottomMenu: {
+            width: "500px",
+            display: "flex",
+            alignItems: "center",
+            justifyContent: "space-between",
+            "& > div": {
+                marginLeft: "auto"
+            }
+        },
+        deleteBtn: {
+            color: red[700],
+            flexShrink: 0,
+            width: 55.4,
+        },
+        searchInput: {
+            position: "absolute",
+            top: "42px",
+            right: "25px",
+        },
+    })();
+
+    const handleRemoveElement = (index: number) => {
+        const list = getContainer();
+        list.splice(index, 1);
+        setContainer(list);
+
+    }
+    const selectElement = (index: number) => {
+        history.push({
+            pathname: createEditPathname,
+            state: { index: index }
+        });
+    }
+
+    const handleSearch = (e: ChangeEvent<HTMLInputElement>) => {
+        let { value } = e.target;
+        setFilterFn({
+            fn: (items: IndexedElement[]) => {
+                if (value === "")
+                    return items
+                else
+                    return items.filter(x => x.element.title.toLowerCase().includes(value));
+            }
+        })
+    }
+    const addIndexes = (item: TitleType, i: number) => {
+        let q: IndexedElement = { index: i, element: item };
+        return q;
+    }
+
+    const onImport = (e: ProgressEvent<FileReader>) => {
+        if (e.target?.result != null) {
+            let jsonString = e.target.result as string;
+            setContainer([...getContainer(), ...JSON.parse(jsonString)]);
+        }
+    }
+
+    return (
+        <div className={classes.root}>
+
+            <Card className={classes.cardWrapper}>
+                <CardHeader
+                    title={`Lista ${listElements}`}
+                    subheader={`${filterFn.fn(getContainer().map(addIndexes)).length} ${listElements}`}
+                />
+                <TextField
+                    className={classes.searchInput}
+                    onChange={handleSearch}
+                    InputProps={{
+                        startAdornment: (
+                            <InputAdornment position="start">
+                                <Search />
+                            </InputAdornment>
+                        )
+                    }}
+                />
+                <List className={classes.list}>
+                    <Divider />
+                    {filterFn.fn(getContainer().map(addIndexes)).map((item) => ([
```

Comment:
This naming of the function inside of the state object is weird. You should either extract the filtering function to the separate variable or remove "Fn" from the state object name. 
".fn" method name isn't clear either, so giving it a name related to the action of starting of filtration would be nice.